bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/RetryOptions.java

```diff
@@ -54,7 +54,8 @@
       Status.Code.DEADLINE_EXCEEDED,
       Status.Code.INTERNAL,
       Status.Code.UNAVAILABLE,
-      Status.Code.ABORTED);
+      Status.Code.ABORTED,
```

Comment:
These changes seem like a useful feature outside of any client auth interceptor changes.  Can you please make a PR for just this change + tests?